compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/main/DefaultIOSMain.uikit.kt

```diff
@@ -16,12 +16,28 @@
 
 package androidx.compose.ui.main
 
-import kotlinx.cinterop.*
-import platform.UIKit.*
-import platform.Foundation.*
+import kotlinx.cinterop.ObjCObjectBase
+import kotlinx.cinterop.autoreleasepool
+import kotlinx.cinterop.cstr
+import kotlinx.cinterop.memScoped
+import kotlinx.cinterop.toCValues
+import platform.Foundation.NSStringFromClass
+import platform.UIKit.UIApplication
+import platform.UIKit.UIApplicationDelegateProtocol
+import platform.UIKit.UIApplicationDelegateProtocolMeta
+import platform.UIKit.UIApplicationMain
+import platform.UIKit.UIResponder
+import platform.UIKit.UIResponderMeta
+import platform.UIKit.UIScreen
+import platform.UIKit.UIViewController
+import platform.UIKit.UIWindow
 
 private var _rootViewController: UIViewController? = null
 
+@Deprecated(
+    "Crashes app under some circumstances. Will be removed in the upcoming release.",
```

Comment:
I'd rather mention that it wasn't supposed to be a public API. Mention of a crash in the deprecation message doesn't sound good

Reply:
Done